### PR TITLE
Add Particle Network AA Guide

### PR DESCRIPTION
**Pull request by TABASCOatw**
**What changed? Why?**

This PR adds a detailed guide diving into the process of leveraging Particle Network's Smart Wallet-as-a-Service (built-in account abstraction stack working with MPC-TSS social logins) within a demo application built on Base.

Particle Network has been deployed on Base for quite some time, with its Bundler and Omnichain Paymaster (alongside core compatibility with providers such as Biconomy) recently being added. Specifically, this guide has a reader create an application that onboards a user into a smart account (either SimpleAccount, Light Account, Biconomy, or CyberConnect) on Base Sepolia through social login, which then executes a gasless (sponsored) transaction.

**Notes to reviewers**

Feel free to suggest any changes in wording, technical details, etc. - open to switching things around a bit.

**Does this PR add a new token to the bridge?**

- [x] No, this PR does not add a new token to the bridge
- [ ] I've confirmed this token doesn't use a bridge override
- [ ] I've confirmed this token is an OptimismMintableERC20

Please include evidence of both confirmations above for your reviewers.

### CODE REVIEW

File: apps/base-docs/docs/building-with-base/guides/account-abstraction-with-particle-network.md [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+
+This document will guide you through the process of using Particle Network within your Base application, building a simple React project using `create-react-app`, Particle Auth Core, and Particle's AA SDK.
+
+---
+
+## Objectives
+
+By the end of this guide, you should be able to:
+
+- Use Particle Auth Core to generate an Externally-Owned-Account (EOA) via a social login.
```
Comment (briandoyle81CB) [inline apps/base-docs/docs/building-with-base/guides/account-abstraction-with-particle-network.md:34]: Nit, but can you please remove the periods from the end of the LO bullets?

File: apps/base-docs/docs/building-with-base/guides/account-abstraction-with-particle-network.md [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+
+WaaS providers other than Particle Network include [Web3Auth](https://web3auth.io), [Privy](https://docs.base.org/building-with-base/guides/account-abstraction/intro-to-privy), and [Magic](https://magic.link/), among others.
+
+### Account Abstraction
+
+Particle Network also aims to tackle the second friction point described above: account flexibility.
+
+Account Abstraction refers to a transition away from standard account structures, EOAs, to smart accounts. Smart accounts are contracts that act as a wallet, providing users with an account that feels equivalent to an EOA but is intrinsically programmable (due to it being a smart contract) and thus more flexible.
+
+The most popular modern implementation of Account Abstraction is [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337), which enables Account Abstraction without any consensus-layer protocol changes. It does this through numerous components of supporting infrastructure, including a Bundler and Paymaster.
```
Comment (briandoyle81CB) [inline apps/base-docs/docs/building-with-base/guides/account-abstraction-with-particle-network.md:81]: Another nit, but do you mind changing to reference-style links?

Comment (TABASCOatw) [reply apps/base-docs/docs/building-with-base/guides/account-abstraction-with-particle-network.md]: Sure, no problem! Should be good to go now, let me know if the new version works.

**Comment by zencephalon** [comment]
@TABASCOatw Just a merge conflict to fix now, thank U!

**Comment by TABASCOatw** [comment]
> @TABASCOatw Just a merge conflict to fix now, thank U!

Sounds good! Should be all good to go now 🤝

**Comment by zencephalon** [comment]
Could U check out these build failures? 😅

**Comment by TABASCOatw** [comment]
> Could U check out these build failures? 😅

All fixed, had some structural updates that needed to happen

**Review by zencephalon — APPROVED** [review]

**Pull request merged**